quay-enterprise/insert-custom-cert.md

```diff
@@ -0,0 +1,41 @@
+# Add Certificate to the Quay Enterprise Container
+
+In the config directory create the a new directory `extra_ca_certs/` and copy any custom certs required to this directory. On boot Quay Enterprise will update the list of trusted certificates via the `update-ca-certificates` command. 
```

Comment:
> On boot Quay Enterprise will update the list of trusted certificates via the `update-ca-certificates` command. 

This is an implementation detail of Quay being on a Debian-based base-image. If we were to ever move this sentence would cease to be true. I think it's unnecessary.